3_visualize.R

```diff
@@ -93,6 +93,17 @@ p3_targets_list <- list(
                     sites_crs = 4269),
     format = "file"
   ),
+  # with synoptic sampling sites
+  tar_target(
+    p3_nhdv2_attr_syn_sites_png,
+    plot_nhdv2_attr(attr_data = p2_nhdv2_attr_refined %>% select(-hru_segment),
+                    network_geometry = p1_reaches_sf,
+                    file_path = "3_visualize/out/nhdv2_attr_png/synoptic",
+                    plot_sites = TRUE, 
+                    sites = p2_syn_sites_w_segs, 
```

Comment:
Should `sites` here refer to `p2_flow_sites_w_segs`? I haven't run the pipeline, but I'm not seeing where the new flow sites are getting plotted.

Reply:
I think I just assumed that one of the png targets would correspond with the flow sites. If we're only interested in the csv file containing the flow gages located in high road density/NPDES catchments, we can leave it there. Whether we create a separate plotting target in this branch for the flow sites might depend on how often we'll revisit this task of identifying synoptic sites that also have streamflow data. I don't really have a strong preference.

Reply:
OK, sounds good to me.